Modules/Microsoft365DSC/Examples/Resources/AADApplication/1-ConfigureAADApplication.ps1

```diff
@@ -53,7 +53,7 @@ Configuration Example
                 }
             )
             Ensure                     = "Present"
-            GlobalAdminAccount         = $credsGlobalAdmin
+            Credential         = $credsGlobalAdmin
```

Comment:
Did you use Search/Replace......indentation is incorrect in all examples ;-)

Reply:
Fixed all example identation